integration_tests/suite/helpers/database.py

```diff
@@ -139,6 +167,11 @@ def insert_agent(self, **kwargs):
                 'language': agent.language,
             }
 
+    def insert_queue_feature(self, **kwargs):
```

Comment:
```suggestion
    def insert_queue(self, **kwargs):
```
I understand why you write queue_features, but it's a technical implementation detail of DB to have inheritence with a table called `queue`. Outside DB, we can talk of `queue` without mention of `feature` (same logic for user, group, line, etc..)

Reply:
Ok, I only named it as such because I was calling `add_queuefeature` on the "Inserter". There was also an `add_queue` so I didn't want to make it confusing in case it was used in future, but if you think it won't be used I'll rename it.

Reply:
arg ... you're right, `add_queuefeature` won't add queue by default. So yes you should leave `feature` suffix :vomiting_face:

Reply:
Or if you can remove it if you're willing to wait if you really need `add_queue` (I don't think, because `Queue` should be really only for asterisk, and not for logic inside agid ... anyways I don't know) :cowboy_hat_face:

Reply:
Haha, ok. Well, I'll do some more tests and if I don't use it by the time I stop, I'll refactor it then.

Reply:
Ok, I renamed them. I did not end up having to use `add_queue` after all, but it was close.